python/mleap/sklearn/base.py

```diff
@@ -58,11 +58,18 @@ def set_input_features(transformer, input_features):
         transformer.input_features = input_features
 
     def serialize_to_bundle(self, transformer, path, model_name):
+        if transformer.normalize:
+            raise ValueError("MLeap linear regression does not support normalization")
 
-        # compile tuples of model attributes to serialize
         attributes = list()
-        attributes.append(('intercept', transformer.intercept_.tolist()[0]))
-        attributes.append(('coefficients', transformer.coef_.tolist()[0]))
+        if len(transformer.coef_.shape) == 2 and transformer.coef_.shape[0] == 1:
```

Comment:
`If multiple targets are passed during the fit (y 2D), this is a 2D array of shape (n_targets, n_features), while if only one target is passed, this is a 1D array of length n_features.`
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html

Reply:
You can fit a linear regressor on a `y` array with shape `(n,1)`, so we still have to consider a 2D `transformer.coef_` array that only denotes one target variable